.rubocop_todo.yml

```diff
@@ -141,6 +141,10 @@ Rails/HelperInstanceVariable:
   Exclude:
     - 'app/helpers/title_helper.rb'
 
+Rails/LexicallyScopedActionFilter:
+  Exclude:
+    - 'app/controllers/oauth2_applications_controller.rb'
```

Comment:
Alternatively, we could add something like

```
def show
  super
end
```

for each of the affected methods

Reply:
I tried that ;-) The problem is that just triggers a different rubocop warning:

```
app/controllers/oauth2_applications_controller.rb:14:3: W: [Correctable] Lint/UselessMethodDefinition: Useless method definition detected.
  def show ...
  ^^^^^^^^
```